accounts/abi/bind/precompilebind/precompile_config_template.go

```diff
@@ -15,44 +15,48 @@ const tmplSourcePrecompileConfigGo = `
 // See the tutorial in https://docs.avax.network/subnets/hello-world-precompile-tutorial for more information about precompile development.
 
 /* General guidelines for precompile development:
-1- Read the comment and set a suitable contract address in generated module.go. E.g:
+1- Set a suitable config key in generated module.go. E.g: "yourPrecompileConfig"
+2- Read the comment and set a suitable contract address in generated module.go. E.g:
 	ContractAddress = common.HexToAddress("ASUITABLEHEXADDRESS")
-2- Set a suitable config key in generated module.go. E.g: "yourPrecompileConfig"
 3- It is recommended to only modify code in the highlighted areas marked with "CUSTOM CODE STARTS HERE". Typically, custom codes are required in only those areas.
 Modifying code outside of these areas should be done with caution and with a deep understanding of how these changes may impact the EVM.
 4- Set gas costs in generated contract.go
-5- Add your config unit tests under generated package config_test.go
-6- Add your contract unit tests undertgenerated package contract_test.go
-7- Additionally you can add a full-fledged VM test for your precompile under plugin/vm/vm_test.go. See existing precompile tests for examples.
-8- Add your solidity interface and test contract to contract-examples/contracts
-9- Write solidity tests for your precompile in contract-examples/test
-10- Create your genesis with your precompile enabled in tests/e2e/genesis/
-11- Create e2e test for your solidity test in tests/e2e/solidity/suites.go
-12- Run your e2e precompile Solidity tests with 'E2E=true ./scripts/run.sh'
+5- Force import your precompile package in precompile/registry/registry.go
+6- Add your config unit tests under generated package config_test.go
+7- Add your contract unit tests undertgenerated package contract_test.go
```

Comment:
```suggestion
7- Add your contract unit tests under the generated package contract_test.go
```

Reply:
Would it make sense to move the instructions completely out of the code and into a separate README instead? Out of scope for this PR, but think this would be cleaner

Reply:
Same comment on the typo in the other places where this is copy-pasted

Reply:
yea I just wanted to make sure they see this before modifying ANY code at all, but having the same comment in each files looks a little bit ugly.